dapp/src/helpers/discord.ts

```diff
@@ -12,15 +12,16 @@ export const discordMessageUrlToIds = (url: string): string[] => {
 export const verifyDiscord = async (
   userData,
   discordHandle,
-  discordMessageUrl
+  discordMessageUrl,
+  pkh
 ) => {
   let [channelId, messageId] = discordMessageUrlToIds(discordMessageUrl);
 
   try {
     let res: any = await fetch(
       `${witnessUrl}/witness_discord?pk=${
         userData.account.publicKey
-      }&channelId=${channelId}&messageId=${messageId}&discordHandle=${encodeURIComponent(discordHandle)}`
+      }&channelId=${channelId}&messageId=${messageId}&discordHandle=${encodeURIComponent(discordHandle)}&pkh=${pkh}`
```

Comment:
I think it's a fair assumption that the PKH can be deduced for all kinds of PKs? `blakesig::hash_public_key(&jwk)` for Tezos

Reply:
I'll update to just use the hash presented PK when we unify the implementation. This is more of a quick fix.

Reply:
Ended up closing this loose end.